Containerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:22.04
+
+ENV debian_frontend=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    libncurses5-dev \
+    libncursesw5-dev \
+    git \
+    gcc \
+    make \
+    zlib1g-dev \
+    libssl-dev \ 
+    libffi-dev \ 
+    libbz2-dev
```

Comment:
Are all of these necessary?

Reply:
You're welcome ☺️

```bash
zlib1g-dev 
libssl-dev  
libffi-dev 
libbz2-dev
```

These libraries are required by `pyenv` which I used to install `python 3.7` in the Container (their [Dockerfile](https://github.com/pyenv/pyenv/blob/v2.4.0/Dockerfile)).

I don't have any particular reason for `makefile` vs `Makefile` and the only reason to use the capitalised version would just to have it [appear higher in the file list of a directory](https://www.gnu.org/software/make/manual/make.html#Makefile-Names). If you'd prefer, I can update the filename. 

The choice of using `Containerfile` is, I suppose, by choice and habit. I will switch it to a `Dockerfile` for convenience. I typically use a [`Taskfile.yml`](https://taskfile.dev) to abstract a lot of my commands (which reflects the simpler lower-cased `makefile`) but didn't include that as it isn't a commonly used tool.